frontend/src/components/LandingPage/LandingPage.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react'
+import Tools from './Tools/Tools'
 //Import components that you create here
 
 function LandingPage(){
     return(
         //Use components that you import here
-        <></>
+        <div id='tools'>
```

Comment:
Just out of curiosity, is the id used?

Reply:
It is not, but I put it there in case we want to style the landing page in the future

Reply:
> Hey Jerry, I left one comment but I wanted to add something else as well. What do you think about moving the images to the public folder?

Yep I can do that

Reply:
Since it has no functionality, what do you think about removing it?